src/js/actions/shortcuts.js

```diff
@@ -158,6 +161,31 @@ define(function (require, exports) {
     removeShortcut.modal = true;
 
     /**
+     * Executes a given shortcut given an id of some sort
+     * 
+     * @param {string} id of shortcut clicked
+     * @return {Promise}
+     */
+    var executeShortcut = function (id) {
+        var shortcutStore = this.flux.store("shortcut"),
+            shortcuts = shortcutStore.getState().shortcuts;
+
+        if (!shortcuts) {
```

Comment:
Is this checking necessary even though the default value in the shortcut store is an array?